bin/vagrant_provision.sh

```diff
@@ -38,8 +38,9 @@ virtualenv $INSTALL_DIR/virtualenv
 chown -R vagrant $INSTALL_DIR/virtualenv
 source $INSTALL_DIR/virtualenv/bin/activate
 
+pip install -U 'pip<1.7'
```

Comment:
This must be pip<7.0. Pip 7.0 does not work with peep and pip 1.7 is very old version.